dwh/indicators.py

```diff
@@ -79,6 +79,13 @@ def create_args(parser: argparse.ArgumentParser):
       type=str
   )
 
+  parser.add_argument(
+        '--custom_parameters',
```

Comment:
Is this used anywhere in this script? If not, can we drop it from here and docker config and add when needed?

Reply:
Sure!